zerver/lib/bugdown/__init__.py

```diff
@@ -1318,6 +1318,31 @@ def __init__(self, parser: Any) -> None:
         super().__init__(parser)
         parser.markdown.tab_length = 4
 
+class BlockQuoteProcessor(markdown.blockprocessors.BlockQuoteProcessor):
+    """ Process BlockQuotes.
+
+        Based on markdown.blockprocessors.BlockQuoteProcessor, but with 2-space indent
+    """
+
+    # Original regex for blockquote is RE = re.compile(r'(^|\n)[ ]{0,3}>[ ]?(.*)')
+    RE = re.compile(r'(^|\n)(?!(?:[ ]{0,3}>\s*(?:$|\n))*(?:$|\n))'
+                    r'[ ]{0,3}>[ ]?(.*)')
+
+    def clean(self, line: str) -> str:
+        # HACK: Silence all the mentions inside blockquotes while
+        # we also remove the '>' from beginning of each line.
+
+        mention_re = re.compile(mention.find_mentions)
```

Comment:
Either we should compile this on import inside `mention.py` or in a `__init__` sort of thing -- we don't want to be requesting compilation on processing every line of a block quote!